Loading/ru.lproj/InfoPlist.strings

```diff
@@ -1 +1 @@
-CFBundleName = "Загрузка";
CFBundleDisplayName = "Загрузка";
NSHumanReadableCopyright = "© 2015 Mike McFadden / Bonzai Программы\nВсе права защищены.";
\ No newline at end of file
+CFBundleName = "Loading";
CFBundleDisplayName = "Loading";
NSHumanReadableCopyright = "© 2015 Mike McFadden / Bonzai Apps\nВсе права защищены.";
```

Comment:
You're probably right about the translated product names – I know Apple doesn't translate the name "Finder", although they do translate "Calendar" and "Notes" so I wasn't sure which way I'm supposed to handle it. How does it work if someone doesn't speak English, though? I'm completely clueless about this if you couldn't tell.

For the company name I wasn't 100% sure and had to look it up when I was making the website. I tend to use Bonzai as the trademark part, while Apps would be a generic description. Before this I had Bonzai Games, for example. I dunno.